manifest.json

```diff
@@ -32,7 +32,10 @@
   },
 
   "background": {
-    "scripts": ["background.js"]
+    "scripts": [
+      "node_modules/webextension-polyfill/dist/browser-polyfill.min.js",
```

Comment:
Good idea :) The promise-based `browser.*` API is so much nicer, will be good to not have to hamstring ourselves to `chrome.*`.

Reply:
Ouch! I just noticed I'm including this but still using `chrome` instead of `browser`. Let me fix this in a follow-up PR